darwin/cli_functions.py

```diff
@@ -773,7 +773,7 @@ def dataset_import(
     delete_for_empty: bool = False,
     import_annotators: bool = False,
     import_reviewers: bool = False,
```

Comment:
This just removes the last place where using multi CPU defaults on.  Will need changing back when we re-enable it.